### PR TITLE
[11.0][MIG] stock reserve

### PR DESCRIPTION
**Pull request by RodrigoBM**
Standard migration

### CODE REVIEW

**Comment by kikopeiro** [comment]
Seems working OK :+1:

**Comment by liebana** [comment]
LGTM :+1:

File: stock_reserve/model/stock_reserve.py [hunk 1]
```diff
@@ -137,7 +137,7 @@ def release_validity_exceeded(self, ids=None):
         """ Release all the reservation having an exceeded validity date """
         domain = [
             ('date_validity', '<', fields.date.today()),
-            ('state', '=', 'assigned')]
+            ('state', 'in', ('assigned', 'partially_available', 'confirmed'))]
```
Comment (rousseldenis) [inline stock_reserve/model/stock_reserve.py:144]: @RodrigoBM Why changing this ?

**Comment by rousseldenis** [comment]
@RodrigoBM What's the status of this ?

**Pull request closed without merging**